README.linux.md

```diff
@@ -57,31 +57,21 @@
     ./README.CentOS.bash
     ```
 
-## For Ubuntu:
-
-- Install Dependencies
-  When you run the README.ubuntu.bash script for dependencies, you will be asked to configure realm for kerberos.
-  You can enter any realm, since this is just for testing, and during testing, it will reconfigure a local server/client.
-  If you want to skip this manual configuration, use:
-  `export DEBIAN_FRONTEND=noninteractive`
+## For Ubuntu (versions 20.04 or newer):
```

Comment:
Ubuntu 24 has no available package for python2. Need either to limit version to 20 && 22 or provide instruction how to build ancient python2 from source.

Reply:
Got it, now I don’t want to waste time checking compatibility with Ubuntu 24, so I decided to limit the instructions to versions 20.04 and 22.04. In the future, we can return to this issue and add to the instruction.